### PR TITLE
BETA: Register klone.is-a.dev

### PR DESCRIPTION
**Pull request by Vai3soh**
Added `klone.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/klone.json
Comment (Stef-00012) [inline domains/klone.json:7]: I cannot reach your website, keeps on loading

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/klone.json
Comment (Vai3soh) [reply domains/klone.json:7]: > I cannot reach your website, keeps on loading

How was I supposed to get a web site up without having a domain name?
Now I get a different ip address (```68.65.123.44```) so I can't get it up by domain name again: 

```
dig +short klone.is-a.dev
68.65.123.44
```
Go to ```url: http://45.153.69.189```

Comment (Stef-00012) [reply domains/klone.json:7]: now it works, first time i opened the site through the ip too and it just kept loading

Comment (Stef-00012) [reply domains/klone.json:7]: `68.65.123.44` is the default IP that redirects all unclaimed domains to https://www.is-a.dev

Comment (Vai3soh) [reply domains/klone.json:7]: > `68.65.123.44` is the default IP that redirects all unclaimed domains to https://www.is-a.dev

So what's next, who's going to change the A record? How much longer do we wait?

Comment (Stef-00012) [reply domains/klone.json:7]: first you have to wait for your pr to get approved & merged by a maintainer
once it's merged you have to wait for the DNS to propagate

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**
